runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -6242,7 +6242,7 @@ TR::Node *TR_EscapeAnalysis::createConst(TR::Compilation *comp, TR::Node *node,
    if (type.isVector())
       {
       result = TR::Node::create(node, TR::ILOpCode::createVectorOpCode(TR::vsplats, type), 1);
-      result->setAndIncChild(0, TR::Node::create(node, comp->il.opCodeForConst(type), value));
+      result->setAndIncChild(0, TR::Node::create(node, comp->il.opCodeForConst(type.getVectorElementType()), value));
```

Comment:
LGTM. Just curious: which test case caused this code to be executed?

Reply:
I was seeing the assert being thrown while compiling this method https://github.com/ibmruntimes/openj9-openjdk-jdk18/blob/9bdb9425c548ea2917c5b79ecbb5788b86314c00/test/jdk/jdk/incubator/vector/Int128VectorTests.java#L5089-L5098 in BumbleBench